src/frontends/pytorch/src/op/repeat.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_repeat(NodeContext& context) {
+    auto x = context.get_input(0);
+    auto repeats = context.get_input(1);
+    auto one = context.mark_node(opset8::Constant::create(element::i64, Shape{}, {1}));
+    auto sizes_shape = context.mark_node(std::make_shared<opset8::ShapeOf>(repeats, element::i64));
+    auto expand_shape = context.mark_node(std::make_shared<opset8::Broadcast>(one, sizes_shape));
+    auto expanded_input =
+        context.mark_node(std::make_shared<opset8::Broadcast>(x, expand_shape, ov::op::BroadcastType::BIDIRECTIONAL));
+    return {context.mark_node(std::make_shared<opset8::Tile>(expanded_input, repeats))};
```

Comment:
Not sure, but I think it is a good reason to propose `Tile` extension to avoid all these `Broadcast` tricks. I mean unlikely this pattern can really be adjusted from FW to FW and the logic involves dealing with shapes only, so no real heavy computations behind, only shape manipulations which traditionally draws attention because it appears to be slow.

Reply:
Do we have kind of backlog for new ops?